validphys2/src/validphys/closuretest/multiclosure.py

```diff
@@ -229,10 +247,11 @@ def dataset_replica_and_central_diff(
         var_diff_sqrt = var_diff_sqrt.transpose(2, 1, 0)
         central_diff = central_diff.T
 
-    var_diff = var_diff_sqrt ** 2
+    var_diff = var_diff_sqrt**2
     sigma = np.sqrt(var_diff.mean(axis=0))  # sigma is always positive
     return sigma, central_diff
 
+
```

Comment:
remove this line

Reply:
See commit:
[4c2634d](https://github.com/NNPDF/nnpdf/pull/1682/commits/4c2634dee29d8e5733c98b8542cf12baa33ae2cd)